Marlin/src/HAL/DUE/dogm/u8g_com_HAL_DUE_shared_hw_spi.cpp

```diff
@@ -100,11 +99,7 @@ uint8_t u8g_com_HAL_DUE_shared_hw_spi_fn(u8g_t *u8g, uint8_t msg, uint8_t arg_va
 
       spiBegin();
 
-      #ifndef SPI_SPEED
-        #define SPI_SPEED SPI_FULL_SPEED  // use same SPI speed as SD card
-      #endif
-      spiInit(2);
-
+      spiInit(LCD_SPI_SPEED);
```

Comment:
This just replaces the 'mystery number' 2 with something meaningful, defining it at the top. It could just be `SPI_QUARTER_SPEED` instead, but constants like this could be useful to override in development.